backends/tfhe-cuda-backend/cuda/include/pbs/pbs_utilities.h

```diff
@@ -83,23 +83,25 @@ template <typename Torus> struct pbs_buffer<Torus, PBS_TYPE::CLASSICAL> {
   Torus *temp_lwe_array_in;
 
   PBS_VARIANT pbs_variant;
-  bool uses_noise_reduction;
+  PBS_MS_REDUCTION_T
```

Comment:
Looks strange that this is on two lines

Reply:
Not sure what happened here. I will fix it.